lambda/custom/src/handlers/helpers.ts

```diff
@@ -33,22 +33,6 @@ const getSelectedChild: (name: string) => Promise<Child | undefined> = async (na
   }
 };
 
-const getResolvedSlotValue: (requestEnvelope: RequestEnvelope, slotName: string) => string | undefined = (
-  requestEnvelope: RequestEnvelope,
-  slotName: string
-) => {
-  const slot = getSlot(requestEnvelope, slotName);
-  const resolutions = slot.resolutions?.resolutionsPerAuthority;
-  const resolution = resolutions?.find(x => x !== undefined);
-
-  if (resolution?.status.code === 'ER_SUCCESS_MATCH') {
```

Comment:
This function is no longer needed because the validation is being taken care of by Alexa.